Tests/SonarQube.Bootstrapper.Tests/ArgumentProcessorTests.cs

```diff
@@ -30,19 +30,20 @@ public void ArgProc_UnrecognizedArgumentsAreIgnored()
         }
 
         [TestMethod]
-        public void ArgProc_MissingUrl()
+        public void ArgProc_DefaultUrlUsedIfNotSupplied()
         {
             // Arrange
             TestLogger logger = new TestLogger();
 
             // Act
-            IBootstrapperSettings settings = CheckProcessingSucceeds(logger, "/d:SONAR.host.url=foo"); // case-sensitive key name so won't be found
+            IBootstrapperSettings settings = CheckProcessingSucceeds(logger,
+                ArgumentProcessor.BeginVerb,
+                "/d:SONAR.host.url=foo"); // case-sensitive key name so won't be found
 
             // Assert
             Assert.AreEqual(BootstrapperSettings.DefaultHostUrl, settings.SonarQubeUrl, "Expecting the host url to be the default");
-            logger.AssertWarningsLogged(2);
             logger.AssertSingleWarningExists(BootstrapperSettings.DefaultHostUrl); // a warning about the default host url should have been logged
-            logger.AssertSingleWarningExists(ArgumentProcessor.BeginVerb); 
```

Comment:
The purpose of this test is to check handling of URL parameters, not the "begin" verb (and there is already a separate test that checks this).